### ISSUE TITLE
Geobase initiative content modifications

### ISSUE THREAD

**Issue by NatFonSeg** [Canadian-Geospatial-Platform/geo.ca#410]
A lot of changes need to be done to the GeoBase initiative page (https://geo.ca/initiatives/geobase/). I tracked the changes in a word document, but since there is a lot of changes, it might be hard to follow. 

Please let me know if it is ok with you or I provide a clean version without the track changes. 

You will find the image source and new URL in the comments.  

[InitiativeGeoBase.docx](https://github.com/user-attachments/files/17135849/InitiativeGeoBase.docx)
[GeoBaseInitiative.docx](https://github.com/user-attachments/files/17135851/GeoBaseInitiative.docx)

Thank you!

@heamart @Andrea-Merry

**Comment by heamart** [comment]
Thanks! We'll take a look

**Comment by heamart** [comment]
Here are the update pages, some changes may come after review:

ENG: 
[GeoBaseInitiative (2)2.docx](https://github.com/user-attachments/files/17307603/GeoBaseInitiative.2.2.docx)
FRE: 
[InitiativeGeoBase (1).docx](https://github.com/user-attachments/files/17307605/InitiativeGeoBase.1.docx)


@jaredkinger @Andrea-Merry

**Comment by heamart** [comment]
Update to gallery filters titles, they will be the following:
(datasets that are associated with each filter listed under)

EN: 

**Administrative boundaries**
- Aboriginal lands
- Administrative boundaries
- Federal Electoral districts
- Canadian geographical names

**Artificial intelligence**
- GeoAI

**Elevation**
- High resolution digital elevation model
- Canadian digital elevation model
- Canadian digital surface model (2000)

**Geodesy**
- Geodetic control networks

**Hydrography**
- Canadian hydrospatial network
- National hydro network
- GeoAI

**Infrastructure**
- National Road Network
- National Railway Network
- GeoAI

**Land cover**
-  Land cover of Agricultural regions of Canada, circa 2000
- GeoAI

FR:
**Couverture du territoire**

- Couverture de terre des régions agricoles du Canada, vers l’an 2000, index des dates
- GéoIA

**Élévation**

- Modèle numérique d'élévation de haute résolution
- Modèle numérique d'élévation du Canada
- Modèle numérique de surface du Canada, 2000

**Géodésie**

- Réseaux géodésiques

**Hydrographie**

- Réseau hydrospatial canadien
- Réseau hydro national
- GéoIA

**Infrastructure**

- Réseau routier national 
- Réseau ferroviaire national
- GéoIA

**Intelligence artificielle**

- GéoIA

**Limites administrative**

- Limites législatives des terres autochtones du Canada
- Limites administratives au Canada
- Circonscriptions fédérales
- Toponymes canadiens

**Comment by heamart** [comment]
Dataset Gallery changes:

On both the french and english page, the Land cover of Agricultural regions of Canada, circa 2000 dataset needs to be changed. Here is the correct card information:

EN:
[Canadian Land Cover, Circa 2000 (Vector), 1996 to 2005](https://app.geo.ca/result/en/canadian-land-cover,-circa-2000-(vector)---geobase-series,-1996-2005?id=97126362-5a85-4fe0-9dc2-915464cfdbb7&lang=en)
This collection offers land cover products from Landsat satellite imagery.
[Explore ⟶](https://app.geo.ca/result/en/canadian-land-cover,-circa-2000-(vector)---geobase-series,-1996-2005?id=97126362-5a85-4fe0-9dc2-915464cfdbb7&lang=en)

FR:
[Couverture terrestre du Canada, circa 2000 (vectoriel), 1996 à 2005](https://app.geo.ca/result/fr/couverture-du-sol-du-canada,-circa-2000-(vectoriel)---s%C3%A9rie-g%C3%A9obase,-1996-2005?id=97126362-5a85-4fe0-9dc2-915464cfdbb7&lang=fr)
Cette collection inclut des produits de couverture terrestre provenant des observations des capteurs Landsat.
[Explorer ⟶](https://app.geo.ca/result/fr/couverture-du-sol-du-canada,-circa-2000-(vectoriel)---s%C3%A9rie-g%C3%A9obase,-1996-2005?id=97126362-5a85-4fe0-9dc2-915464cfdbb7&lang=fr)

**Comment by jaredkinger** [comment]
Hi @heamart I've updated the [English](https://wordpress.dev.geo.ca/initiatives/geobase/) and [French](https://wordpress.dev.geo.ca/fr/initiatives-fr/geobase/) Geobase pages. Please review and let me know if I missed anything or if there are any formatting changes needed.

**Comment by heamart** [comment]
Looks good! Just waiting on final content owner review for changes

**Comment by heamart** [comment]
good to go
